emboss-format

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Benjamin Olmstead
```

Comment:
Should this be Google LLC?

Reply:
I don't believe so, since Google does not own the copyright to my work.  The CLA gives Google a very broad license, but is not copyright assignation.